### ISSUE TITLE
autolinker breaks in certain instances

### ISSUE THREAD

**Issue by stanleycyang** [gregjacobs/Autolinker.js#160]
This comment breaks autolinker:

"<US_Con_SL_RTNS@dell.com\nHe gave me a 1-800 customer care number that I've called twice.  The last time I called, about 3 weeks ago, the customer rep said he would request an expedited response. He gave me a reference number which is  925767619. Thankyou very much for looking into this.\n\nRonald D Brigham\nbrigham@mtnhome.com"

It references this line:

```js
while( ( currentResult = htmlRegex.exec( html ) ) !== null ) {
```

Looks like it hit an infinite loop

**Comment by joaomgcd** [comment]
This text also has the same effect: 
`asdf ouefof<33we oeweofjojfew oeijwffejifjew ojiwjoiwefj iowjefojwe iofjw:)<33`

**Comment by gregjacobs** [comment]
Interesting analysis @joaomgcd. Must be something exponential in the regex engine. I'll see what I can tweak to get this to work

**Comment by gregjacobs** [comment]
I finally figured this out! Thanks for the great input string examples and analysis guys. 

Seems that Chrome has a weird issue with the possibility of zero whitespace characters between attributes in html, and the regex was thinking that `<3` and `<US_Con_SL_RTNS@dell.com` started an html string. Added a fix that seems to have solved this: 072d104b363f68c5282e5aa93dcec6fb90c83b3d

Fix is in v0.27.0
